src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/special/interactivemaps/InteractiveMapPageObject.java

```diff
@@ -276,6 +276,13 @@ public void verifyMapOpened() {
     PageObjectLogging.log("verifyMapOpened", "Map was opened", true);
   }
 
+  public void verifyMapOpenedForDeleteMapTests() {
+    waitForElementVisibleByElement(mapFrame);
+    driver.switchTo().frame(mapFrame);
+    driver.switchTo().defaultContent();
+    PageObjectLogging.log("verifyMapOpenedForDeleteMapTests", "Map was opened", true);
+  }
```

Comment:
But, you'r not searching for any element after 
```
driver.switchTo().frame(mapFrame);
```
you could just look for iFrame element,w ithout switching